src/assets/sass/protocol/components/forms/_button-container.scss

```diff
@@ -0,0 +1,63 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../includes/lib';
+@import '../../includes/forms/lib';
+
+.mzp-c-button-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin-bottom: $field-v-spacing;
+    text-align: center;
+
+    .mzp-c-button {
+        max-width: $content-xs;
+        width: 100%;
+    }
+
+    // no bottom spacing if it's the last thing in the form
+    .mzp-c-form > .mzp-c-button-container:last-child {
+        margin-bottom: 0;
+    }
+
+    // align center
```

Comment:
If center align is the default do we also need a class for it?

Reply:
It's centered by default on mobile, this makes it centered on all screens. Though if we don't even need that option then we could lose this class, too.

Reply:
I think we should keep the all screens option 👍